pdf/src/content.rs

```diff
@@ -394,7 +394,18 @@ impl OpBuilder {
             }
             "Tf"  => push(Op::TextFont { name: name(&mut args)?, size: number(&mut args)? }),
             "Tj"  => push(Op::TextDraw { text: string(&mut args)? }),
-            "TJ"  => push(Op::TextDrawAdjusted { array: array(&mut args)? }),
+            "TJ"  => {
+                let mut primatives = array(&mut args)?.into_iter();
+                let mut array = Vec::<TextDrawAdjusted>::new();
+
+                while let Some(spacing_or_text) = primatives.next() {
```

Comment:
looks like a or loop to me? `for spacing_or_text in primatives`

Reply:
Whoops, old code leaking through.